modules/ROOT/pages/migration-munit-mock-processor-changes.adoc

```diff
@@ -0,0 +1,71 @@
+= Migrating the Mock Processor (MUnit)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+. The name of the processor has changed from when to mock-when.
+. The attribute messageProcessor has change to processor.
+. In the child node with attribute the attribute name has changed to attributeName.
```

Comment:
In the child node `with` attribute**,** the attribute `name` has changed to `attributeName`.

Reply:
done via suggestion

Reply:
done via suggestion